### DISCUSSION TITLE
Previous tab with multiple pop-ups

### DISCUSSION THREAD

**Discussion by LutraVulgaris** [Clooos/Bubble-Card#981] [Feature requests]
When working with pop-ups in pop-ups it would be handy to just return to the previous pop-upscreen instead of homing.

**Comment by SylvainDx** [comment]
If you have only one source pop-up you can add a close action on the pop-up to retur to his parent
![image](https://github.com/user-attachments/assets/6454cf47-9926-4f41-98f5-bba92836709c)

**Reply by LutraVulgaris** [reply]
Amazing, thanks!
